### PR TITLE
added timeout for last 2 stages of the build

### PR DESCRIPTION
**Pull request by pranavgore09**
added `1 SECONDS` timeout to test if last 2 stages of the build fail within 1 seconds on CI
if yes then I will make it 20 minutes before merging.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@pranavgore09 fabric8/fabric8-ui:SNAPSHOT-PR-2410-9 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2410-fabric8-planner.badger.fabric8.io

**Comment by pranavgore09** [comment]
build worked, 10m timeout should be enough

**Comment by pranavgore09** [comment]
[test]

**Comment by fabric8cd** [comment]
@pranavgore09 fabric8/fabric8-ui:SNAPSHOT-PR-2410-17 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2410-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@pranavgore09 fabric8/fabric8-ui:SNAPSHOT-PR-2410-23 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2410-fabric8-planner.badger.fabric8.io

File: Jenkinsfile [hunk 1]
```diff
@@ -69,33 +69,35 @@ fabric8UITemplate{
...
-           def message = "@${changeAuthor} ${imageName} is deployed and available for testing at https://${route}"
-           container('clients'){
-               flow.addCommentToPullRequest(message, pr, project)
-           }
-       }
-   }
+    def prj = 'fabric8-ui-'+ env.BRANCH_NAME
+    prj = prj.toLowerCase()
+    def route
+    timeout(time: 10, unit: 'MINUTES') {
```
Comment (jarifibrahim) [inline Jenkinsfile]: @pranavgore09 Is 10 minutes timeout sufficient?

**Comment by jarifibrahim** [comment]
@pranavgore09 Can you please squash/clean your commits?

File: Jenkinsfile (hunk 1 above)
Comment (pranavgore09) [reply Jenkinsfile]: sure! I can set the smaller value by observing avg time from builds. I think 5m will be sufficient too

**Comment by fabric8cd** [comment]
@pranavgore09 fabric8/fabric8-ui:SNAPSHOT-PR-2410-25 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2410-fabric8-planner.badger.fabric8.io

File: Jenkinsfile (hunk 1 above)
Comment (jarifibrahim) [reply Jenkinsfile]: :+1:

**Comment by fabric8cd** [comment]
@pranavgore09 fabric8/fabric8-ui:SNAPSHOT-PR-2410-30 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2410-fabric8-planner.badger.fabric8.io

**Pull request merged**